.gitignore

```diff
@@ -1 +1,129 @@
 frontend/public/env.js
+
+
```

Comment:
Do we really need all these?
I tend to put only the necessary gitignores to avoid being bitten by "whoops my legit file wasn't added because it clashes with this 100+ lines gitignore/dockerignore/rsyncignore file"...